src/constants/index.js

```diff
@@ -1,37 +1,43 @@
 export const SupportedNetwork = Object.freeze({
   MAINNET: "Mainnet",
   XDAI: "xDAI",
-  ARBITRUM_ONE: "Arbitrum",
+  ARBITRUM: "Arbitrum",
 });
 
 export const ChainId = Object.freeze({
   [SupportedNetwork.MAINNET]: 1,
   [SupportedNetwork.XDAI]: 100,
-  [SupportedNetwork.ARBITRUM_ONE]: 42161,
+  [SupportedNetwork.ARBITRUM]: 42161,
 });
 
 export const SupportedNetworkForChainId = Object.freeze({
   [ChainId[SupportedNetwork.MAINNET]]: SupportedNetwork.MAINNET,
   [ChainId[SupportedNetwork.XDAI]]: SupportedNetwork.XDAI,
-  [ChainId[SupportedNetwork.ARBITRUM_ONE]]: SupportedNetwork.ARBITRUM_ONE,
+  [ChainId[SupportedNetwork.ARBITRUM]]: SupportedNetwork.ARBITRUM,
 });
 
 export const ChainIdForSupportedNetwork = Object.freeze({
   [SupportedNetwork.MAINNET]: ChainId[SupportedNetwork.MAINNET],
   [SupportedNetwork.XDAI]: ChainId[SupportedNetwork.XDAI],
-  [SupportedNetwork.ARBITRUM_ONE]: ChainId[SupportedNetwork.ARBITRUM_ONE],
+  [SupportedNetwork.ARBITRUM]: ChainId[SupportedNetwork.ARBITRUM],
+});
+
+export const NETWORK_SUBGRAPH_URLS = Object.freeze({
+  MAINNET: "luzzif/swapr-mainnet-alpha",
+  XDAI: "luzzif/swapr-xdai",
+  ARBITRUM: "luzzif/swapr-arbitrum-one",
```

Comment:
We need to change this in the next PR, which will feature a new subgraph.

Reply:
yeah I can add issue for it once we decide to do that